libpyvinyl/BaseCalculator.py

```diff
@@ -74,47 +78,40 @@ def __init__(
-        :type name: DataCollection, list or BaseData
-
-        :param output_keys: The key(s) of this calculator's output data. It's a list of `str`s or
-        a single str.
-        :type output_keys: list or str
+        :param input: The input of this calculator. It can be a `DataCollection`,
+                     a list of `DataCollection`s or a single Data Object.
 
-        :param output_data_types: The data type(s), i.e., classes, of each output. It's a list of the
-        data classes or a single data class. The available data classes are based on `BaseData`.
-        :type output_data_types: list or DataClass
+        :param output_keys: The key(s) of this calculator's output data.
 
-        :param output_filenames: The name(s) of the output file(s). It can be a str of a filename or
-        a list of filenames. If the mapping is dict mapping, the name is `None`. Defaults to None.
-        :type output_filenames: list or str
+        :param output_data_types: The data type(s), i.e., classes, of each output.
+                                  It's a list of the data classes or a single data class.
+                                  The available data classes are based on `BaseData`.
+        :param output_filenames: The name(s) of the output file(s).
+                                 It can be a str of a filename or a list of filenames.
+                                 If the mapping is dict mapping, the name is `None`.
+                                 Defaults to None.
 
-        :param instrument_base_dir: The base directory for the instrument to which this calculator
-        belongs. Defaults to "./". The final exact output file path depends on `instrument_base_dir`
-        and `calculator_base_dir`: `instrument_base_dir`/`calculator_base_dir`/filename
-        :type instrument_base_dir: str
+        :param instrument_base_dir: The base directory for the instrument to which
+                                    this calculator belongs.
+                                    The final exact output file path depends on `instrument_base_dir` and `calculator_base_dir`: `instrument_base_dir`/`calculator_base_dir`/filename
 
-        :param calculator_base_dir: The base directory for this calculator. Defaults to "./". The final
-        exact output file path depends on `instrument_base_dir` and
-        `calculator_base_dir`: `instrument_base_dir`/`calculator_base_dir`/filename
-        :type instrument_base_dir: str
+        :param calculator_base_dir: The base directory for this calculator. The final
+                                    exact output file path depends on `instrument_base_dir` and
+                                    `calculator_base_dir`: `instrument_base_dir`/`calculator_base_dir`/filename
 
         :param parameters: The parameters for this calculator.
-        :type  parameters: Parameters
 
         """
         # Initialize the variables
-        self.__name = None
+        self.__name = ""
```

Comment:
I don't know if it's the standard way to initialize a variable with an empty string. Maybe @CFGrote can comment on that?

Reply:
Ok